Makefile

```diff
@@ -198,6 +206,13 @@ else
 	$(warning skipping Go tests; unable to find Go compiler)
 endif
 
+rust-test:
+ifneq (, $(shell which cargo))
+	cat benchmarks/*.fpcore tests/*.fpcore | racket infra/test-core2rust.rkt -s --error 150
+else
```

Comment:
I think the error flag should be dropped to 3 since except for a handful of operators, Rust uses the system libm. Not thrilled about the `-s` flag but until `asinh` and `acosh` are fixed, it'll do for now.